arangod/Cluster/MaintenanceFeature.cpp

```diff
@@ -649,20 +645,6 @@ arangodb::Result MaintenanceFeature::storeIndexError(
   return Result();
 }
 
-arangodb::Result MaintenanceFeature::indexErrors(
```

Comment:
This was unused code? Isn't that a problem?

Reply:
Problem solved by removing the code :smile:
Not sure what it was originally intended for. However, index errors are still captured and processed when the `copyAllErrors` method is called. Probably someone had plans to access just index errors separately at some point.